### DISCUSSION TITLE
关于chatgpt隔一段时间就失联的问题

### DISCUSSION THREAD

**Discussion by lll123github** [bannedbook/fanqiang#779] [Q&A]
想要问一下使用SSR翻墙的时候会出现chatgpt过了一段时间就会失联的情况，得重新刷新页面。问一下是隔段时间更新转发ip地址是么？如果是的话问一下应该如何解决这个问题呢

**Comment by caizi1288** [comment]
同问，用clashX也是一样问题，另外最近几天似乎直接不能访问了chatGPT了，不知道是账号被封了，还是所有的IP都封了。

**Comment by bannedbook** [comment]
chatgpt过了一段时间就会失联的情况，得重新刷新页面
这个问题，买chatgpt plus 可以解决

**Comment by bannedbook** [comment]
彻底打不开的，购买付费的翻墙服务可以解决

**Reply by caizi1288** [reply]
那就是还是节点被封了，搞的以为账号被识别为CN区域，直接封了呢
